modulemd/v2/modulemd-module-stream.c

```diff
@@ -772,3 +772,69 @@ modulemd_module_stream_get_translation_entry (ModulemdModuleStream *self,
   return modulemd_translation_get_translation_entry (priv->translation,
                                                      locale);
 }
+
+
+gboolean
+modulemd_module_stream_emit_yaml_base (ModulemdModuleStream *self,
+                                       yaml_emitter_t *emitter,
+                                       GError **error)
+{
+  MODULEMD_INIT_TRACE ();
+  g_autofree gchar *version_string = NULL;
+
+  version_string =
+    g_strdup_printf ("%" PRIu64, modulemd_module_stream_get_version (self));
+
+  /* Emit document headers */
+  if (!modulemd_yaml_emit_document_headers (
+        emitter,
+        MODULEMD_YAML_DOC_MODULESTREAM,
+        modulemd_module_stream_get_mdversion (self),
+        error))
+    return FALSE;
+
+  /* Start data: */
+  if (!mmd_emitter_start_mapping (emitter, YAML_BLOCK_MAPPING_STYLE, error))
```

Comment:
Since you wrote the nice macros for `modulemd_module_stream_v1_emit_yaml()`, would you mind using them here as well?